frontend/src/components/common/BulkPermissionResults.vue

```diff
@@ -1,21 +1,101 @@
 <script setup lang="ts">
-import { Dialog } from '@/lib/primevue';
+import { ref } from 'vue';
+import { Column, DataTable, Dialog } from '@/lib/primevue';
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome';
+
 // Props
 type Props = {
   results: Array<Object>;
   resource: any;
   resourceType: string;
 };
-const props = withDefaults(defineProps<Props>(), {});
+
+const props = withDefaults(defineProps<Props>(), {
+  results: undefined
+});
 
 const modelValue = defineModel<boolean>({ default: false });
+
+// Exports DataTable results
+const batchResults = ref();
+
+const exportCSV = (event: any) => {
+  batchResults.value.exportCSV();
+};
 </script>
 <template>
   <Dialog
     v-model:visible="modelValue"
     header="Results"
     :modal="true"
   >
-    <pre>{{ props.results }}</pre>
+    <DataTable
+      ref="batchResults"
+      :export-filename="`${resource.name}_bulk_results`"
+      :value="props.results"
+      class="p-datatable-striped"
+    >
+      <div class="action-buttons">
+        <Button
+          v-tooltip.bottom="'Save results'"
+          aria-label="Save results"
+          class="p-button"
+          @click="exportCSV($event)"
+        >
+          <font-awesome-icon
+            icon="fa-download"
+            class="pl-1 pr-1 pt-1 pb-1"
+          />
+        </Button>
+      </div>
+      <Column
+        field="email"
+        header="Email"
+      />
+      <Column
+        field="description"
+        header="Result"
+      >
+        <template #body="{ data }">
+          <span>
+            <span class="m-1">
+              <font-awesome-icon
+                v-if="data.status === 1"
+                icon="fa-circle-check"
+                class="icon-success"
+              />
+              <font-awesome-icon
+                v-else-if="data.status === 0"
+                icon="fa-circle-minus"
+                class="icon-noaction"
+              />
+              <!--
```

Comment:
not sue if we need to include this comment. but if you want to leave it in that's fine by me. 
technically CHES can in theory return an error (for example if CHES is down or we arent authorized) that we can show (but it will occur for all invites).

Reply:
True, the "further comments" for this PR are probably sufficient.